fcrepo-event-serialization/src/main/java/org/fcrepo/event/serialization/JsonLDEventMessage.java

```diff
@@ -122,53 +65,50 @@ public Agent(final String type, final String name) {
     @JsonProperty("type")
     public List<String> type;
 
-    @JsonProperty("isPartOf")
-    public String isPartOf;
+    @JsonProperty("name")
+    public String name;
 
-    @JsonProperty("wasGeneratedBy")
-    public Activity wasGeneratedBy;
+    @JsonProperty("actor")
+    public String actor;
 
-    @JsonProperty("wasAttributedTo")
-    public List<Agent> wasAttributedTo;
+    @JsonProperty("object")
+    public Object object;
 
     @JsonProperty("@context")
-    public Context context = new Context();
+    public String context;
 
     /**
      * Populate a JsonLDEventMessage from a FedoraEvent
+     * 
      * @param evt The Fedora event
      * @return a JsonLDEventMessage
      */
     public static JsonLDEventMessage from(final FedoraEvent evt) {
         final JsonLDEventMessage msg = new JsonLDEventMessage();
-
+        msg.context = ACTIVITYSTREAMS_NS;
         final String baseUrl = evt.getInfo().get(BASE_URL);
-        final String userAgent = evt.getInfo().get(USER_AGENT);
-
-        final List<Agent> agents = new ArrayList<>();
-        agents.add(new Agent(PROV_NAMESPACE + "Person", evt.getUserID()));
-        if (userAgent != null) {
-            agents.add(new Agent(PROV_NAMESPACE + "SoftwareAgent", userAgent));
-        }
-        msg.wasAttributedTo = agents;
-
-        final List<String> types = evt.getTypes().stream().map(rdfType -> rdfType.getType()).collect(toList());
-        if (!types.contains(PROV_NAMESPACE + "Activity")) {
-            types.add(PROV_NAMESPACE + "Activity");
-        }
-        msg.wasGeneratedBy = new Activity(evt.getEventID(), evt.getDate(), types);
-
-        if (baseUrl == null) {
-            msg.id = "info:fedora" + evt.getPath();
-        } else {
-            msg.isPartOf = baseUrl;
```

Comment:
It's OK to keep an `isPartOf` in the message, as is illustrated in the [basic example](https://fcrepo.github.io/fcrepo-specification/#basic-notification-example).  I wish this was in the spec as a `SHOULD`.   Your call @dbernstein, we could also just create another ticket to add this in if it makes sense in principle to do so, but there are questions, etc

Reply:
@birkland : Leaving it in makes sense.  I'll do that.